inventory_refresh.gemspec

```diff
@@ -33,4 +32,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "simplecov"
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
Haven't seen this one before.... I'm all for 2fa for rubygems not sure we want to require it yet though @Fryguy ?